system-monitor/README.md

```diff
@@ -0,0 +1,41 @@
+# System monitor
+
+Python program monitoring docker container CPU and MEMORY usage.
+
+## Load env
+
+```bash
+conda env create -f env.yml
+conda activate env
+```
+
+## Run monitor
+
+```bash
+python system-monitor.py
+```
+
+## Start monitoring
+Send ```POST``` request to endpoint ```http://localhost:8063/monitor```. Pass the container name to monitor and function name into request body:
+```json
+{
+     "container_name": 'postgres',
+     "monitor": true,
+     "function_name": 'average'
+}
+```
+## Stop monitoring
+Send ```DELETE``` request to ```http://localhost:8063/monitor```. Request body:
+
+```json
+{
+    "monitor": false
+}
+```
+
+## Results
+Monitoring result is saved to file ```average_timestamp.csv```.

```

Comment:
it would be nice to tell what are these numbers in `CPU` and `RAM` columns. Is it percentage usage?